pkgs/top-level/splice.nix

```diff
@@ -164,9 +164,50 @@ in
 {
   inherit splicePackages;
 
-  # We use `callPackage' to be able to omit function arguments that can be
-  # obtained `pkgs` or `buildPackages` and their `xorg` package sets. Use
-  # `newScope' for sets of packages in `pkgs' (see e.g. `gnome' below).
+  /**
+    We use `callPackage' to be able to omit function arguments that can be obtained `pkgs` or `buildPackages` and their `xorg` package sets. Use `newScope' for sets of packages in `pkgs' (see e.g. `gnome' below).
+
+    # Inputs (positional)
+
+    - `funcOrPath` - `(AttrSet -> a) | Path`
```

Comment:
I think the convention was 

- 'name' ( type) : description

Which was already used in.other places in this pr and in lib.